File: rallye-core-service/src/main/java/fr/vandriessche/rallyeschema/coreservice/entities/SubmittedFormSummary.java
package fr.vandriessche.rallyeschema.coreservice.entities;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class SubmittedFormSummary {
	private String id;
	private Integer challenge;
	private Integer page;
	private Integer team;
	private String processingStatus;
	private String processingError;
	private Boolean manualReviewRequired;
	private boolean availableForVerification;
	private boolean leasedByCurrentDevice;
}

File: rallye-webapp/src/app/form-processing/models/submitted-form-summary.ts
export interface SubmittedFormSummary {
  id: string;
  challenge?: number;
  page?: number;
  team?: number;
  processingStatus?: string;
  processingError?: string;
  manualReviewRequired?: boolean;
  availableForVerification: boolean;
  leasedByCurrentDevice: boolean;
}

export interface SubmittedFormSummaryPage {
  content: SubmittedFormSummary[];
  totalElements: number;
  totalPages: number;
  number: number;
  size: number;
}
